OpenEdXMobile/res/values/styles.xml

```diff
@@ -559,6 +559,13 @@
         <item name="android:windowExitAnimation">@anim/slide_down</item>
     </style>
 
+    <style name="CourseTabItem">
+        <item name="android:textSize">@dimen/edx_small</item>
+        <item name="lineHeight">@dimen/widget_margin_double</item>
```

Comment:
`lineHeight` is not effecting the scrollable tabs item. batter to remove it. Also, we already applied padding in the layout.